.gitignore

```diff
@@ -3,6 +3,8 @@ __pycache__
 *.egg-info
 logs/
 Translator-Tests/
+build/
```

Comment:
Where does `build/` come from?

Reply:
not sure. that's more common with Javascript builds, right - none here. I don't think it is relevant to this, unless the UI testing is somehow creating it, LOL. I'll try to remove it.